esignet-default.properties

```diff
@@ -448,7 +449,7 @@ mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.domain.url}
    }},\
   'display': {{'name': 'MOSIP Identity Verifiable Credential', \
                 'locale': 'en', \
-                'logo': {'url': '${mosip.esignet.domain.url}/MOSIP_Logo.png',\
+                'logo': {'url': '${mosip.esignet.file.url}/mosip-logo.png',\
```

Comment:
I see the below is not working.
https://esignet.dev.mosip.net/mosip-logo.png

Also can we avoid the esignet in the URL to show the difference ? Is it not possible to access it using https://api.dev.mosip.net/***

Reply:
now it will be https://api.dev.mosip.net/inji/mosip-logo.png